tests/suites/test_suite_pk.function

```diff
@@ -438,6 +438,9 @@ void pk_rsa_verify_ext_test_vec( data_t * message_str, int digest,
     void *options;
     size_t hash_len;
 
+#if defined(MBEDTLS_USE_PSA_CRYPTO)
```

Comment:
I think this block can be replace with `USE_PSA_INIT( );` and the block at the end of the test function with `USE_PSA_DONE( );`